File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        k = 0
        for i in range(len(nums)):
            if i ==0 or nums[i-1] == nums[i]:
                k = k+1 if i==0 else k
                continue
            nums[k] = nums[i]
            k+=1
            #print(f"nums:{nums}, k:{k},i:{i}")
        return k

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.ts
function removeDuplicates(nums: number[]): number {
    let k = 0
    for (let i=0;i<nums.length;i++){
        if (i===0 || nums[i-1]===nums[i]){
            k = i===0 ? k+1: k
            continue 
        }
        nums[k] = nums[i]
        k++
    }
    return k
};